### PR TITLE
Issue #165 - Remove unnecessary operational restrictions on logs

### PR DESCRIPTION
**Pull request by bifurcation**

### CODE REVIEW

File: draft-ietf-trans-rfc6962-bis.md [hunk 1]
```diff
@@ -552,14 +552,12 @@ Tree no later than a fixed amount of time, known as the Maximum Merge Delay
 (MMD), after the issuance of the SCT. Periodically, the log MUST append all its
 new entries to its Merkle Tree and sign the root of the tree.
 
-Log operators MUST NOT impose any conditions on retrieving or sharing data from
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Do I recall incorrectly that the suggestion was to change the MUST to a SHOULD here? 
Overall I agree this is a policy matter, but making a suggestion here (in a normative or an informative manner) seems like a good idea.

**Comment by eranmes** [comment]
looks good overall, one minor comment.

**Comment by eranmes** [comment]
(just to be clear, I'd also be OK with removing this sentence completely, but rather have some suggestion there)

File: draft-ietf-trans-rfc6962-bis.md [hunk 2]
```diff
@@ -552,14 +552,12 @@ Tree no later than a fixed amount of time, known as the Maximum Merge Delay
...
-the log.
-
 ## Accepting Submissions
 
 Before accepting a submitted certificate or precertificate, the log MUST verify
-that it has a valid signature chain to an accepted trust anchor, using the chain
-of intermediate CA certificates provided by the submitter. Logs SHOULD accept
+that it has a valid signature chain to an accepted trust anchor, using the
+chain of intermediate CA certificates provided by the submitter. (This helps
+protect logs from being overloaded by invalid submissions.) Logs SHOULD accept
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: That bit is also explained in the introduction - don't mind replicating it here, but seems a bit strange that there's a whole sentence in braces? 
How about:
using the
"...of intermediate CA certificates provided by the submitter. Logs SHOULD accept		 +chain of intermediate CA certificates provided by the submitter, to help protect logs from being overloaded by invalid submissions".

File: draft-ietf-trans-rfc6962-bis.md [hunk 3]
```diff
@@ -552,14 +552,15 @@ Tree no later than a fixed amount of time, known as the Maximum Merge Delay
...
 
 ## Accepting Submissions
 
-Before accepting a submitted certificate or precertificate, the log MUST verify
-that it has a valid signature chain to an accepted trust anchor, using the chain
-of intermediate CA certificates provided by the submitter. Logs SHOULD accept
+To avoid logs being overloaded by invalid submissions, before accepting a
+submitted certificate or precertificate, the log MUST verify that it has a
+valid signature chain to an accepted trust anchor, using the chain of
+intermediate CA certificates provided by the submitter. Logs SHOULD accept
```
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: "logs...the log" mixes singular and plural unnecessarily.

I suggest rewording this sentence as follows:
"To avoid being overloaded by invalid submissions, the log MUST NOT accept any
submission until it has verified that the submitted certificate or
precertificate has a valid signature chain to an accepted trust anchor, using
the chain of intermediate CA certificates provided by the submitter."

What do you think?

Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: LGTM - very precise about forbidding the log from accepting a submission even if it can build alternative chains, which I like.

**Comment by eranmes** [comment]
Closing in favour of re-working the text in https://github.com/google/certificate-transparency-rfcs/pull/250 (since I've adopted @robstradling 's suggestion, there's no need to merge this PR)
